addons/func_godot/src/netradiant_custom/netradiant_custom_gamepack_config.gd

```diff
@@ -49,6 +49,12 @@ extends Resource
 ## Skip texture path that gets applied to caulk and nodrawnonsolid shaders.
 @export var skip_texture: String = "textures/special/skip"
 
+## Optional variables to include in the build profile.
+@export var build_xml_variables: Dictionary
```

Comment:
Should be `default_build_menu_variables`, since we don't actually use the term `build_xml` anywhere and it's too much of a shorthand. Exposed variables should always be fully descriptive and easily associated with other processes and exports.

Can we also get a more expository description on what these variables are for? Not quite a tutorial, but since NRC is so poorly documented it might be helpful to let NRC users know what this is for.